docs/0005-hashed-n-tuple-id-layout.md

```diff
@@ -0,0 +1,321 @@
+                ├── inventory.json.sha512
+                └── v1 [...]
+```
+
+### Example 2
+
+The example demonstrates the effects of modifying the default parameters to use a different `digestAlgorithm`, smaller `tupleSize`, and a larger `numberOfTuples`.
+
+#### Parameters
+
+```json
+{
+    "digestAlgorithm": "md5",
+    "tupleSize": 2,
+    "numberOfTuples": 15
+}
+```
+
+#### Mappings
+
+| Object ID | Digest | Object Root Path |
+| --- | --- | --- |
+| object-01 | ff75534492485eabb39f86356728884e | `ff/75/53/44/92/48/5e/ab/b3/9f/86/35/67/28/88/object-01` |
+| ..hor/rib:le-$id | 08319766fb6c2935dd175b94267717e0 | `08/31/97/66/fb/6c/29/35/dd/17/5b/94/26/77/17/%2e%2ehor%2frib%3ale-%24id` |
+
+#### Storage Hierarchy
+
+```
+[storage_root]/
+├── 0=ocfl_1.0
+├── ocfl_layout.json
+├── 0005-hashed-n-tuple-id-layout.json
+├── 08/
+│   └── 31/
+│       └── 97/
+│           └── 66/
+│               └── fb/
+│                   └── 6c/
+│                       └── 29/
+│                           └── 35/
+│                               └── dd/
+│                                   └── 17/
+│                                       └── 5b/
+│                                           └── 94/
+│                                               └── 26/
+│                                                   └── 77/
+│                                                       └── 17/
+│                                                           └── %2e%2ehor%2frib%3ale-%24id/
+│                                                               ├── 0=ocfl_object_1.0
+│                                                               ├── inventory.json
+│                                                               ├── inventory.json.sha512
+│                                                               └── v1 [...]
+└── ff/
+    └── 75/
+        └── 53/
+            └── 44/
+                └── 92/
+                    └── 48/
+                        └── 5e/
+                            └── ab/
+                                └── b3/
+                                    └── 9f/
+                                        └── 86/
+                                            └── 35/
+                                                └── 67/
+                                                    └── 28/
+                                                        └── 88/
+                                                            └── object-01/
+                                                                ├── 0=ocfl_object_1.0
+                                                                ├── inventory.json
+                                                                ├── inventory.json.sha512
+                                                                └── v1 [...]
+```
+
+### Example 3
+
+This example demonstrates what happens when `tupleSize` and `numberOfTuples` are set to `0`. This is an edge case and not a recommended configuration.
+
+#### Parameters
+
+```json
+{
```

Comment:
In order to avoid reader confusion, may as well use the parameter referencing format detailed in the top-level readme:
https://github.com/OCFL/extensions#referencing-parameters

Reply:
seems like this could change?